docs/preview/index.html

```diff
@@ -26,10 +26,35 @@
       <div id="header">
         <!--TODO-->
       </div>
-      
-      <div>
-        <video id="video-background" autoplay loop muted playsinline poster="assets/images/poster.jpg">
-          <source src="assets/videos/video-background.mp4" type="video/mp4">
+      <nav id="navbar">
```

Comment:
I suggest a tag nesting like this: 
```
<header id="header" class="active">
    <nav id="navbar">
    </nav>
    <div class="video-wrapper">
    </div>
</header>
```
What do you think? --> **header** tag instead of **section**.
Then we could create separate scss files to header (including nav), sections, footer etc.